extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/NarayanaJtaConfiguration.java

```diff
@@ -22,4 +22,10 @@
      */
     @ConfigItem()
     public Optional<String> xaNodeName;
+
+    /**
+     * The default transaction timeout is seconds
+     */
+    @ConfigItem
+    public Optional<Integer> defaultTransactionTimeout;
```

Comment:
It should be a `Duration`.

And I think I would define the default value here so that it gets documented when we generate the doc.

Reply:
OK, should I re-use the default from Narayana (60s) or configure a more higher default as we often find on some implementation (300s).

Reply:
To me 60 seems fine but ask  on quarkus-dev so Michael can step up.